Updated Assignment2 as a part of Python Session

Bishudsi opened the pull request:
<!-- PULL REQUEST TITLE: UofT-DSI | <Python> - Assignment <2>-->

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)

## What did you learn from the changes you have made?
      usage of arrays, function calls

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
     NA
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
     yes there were challenges to understand the expected output of the assignment, how the function calls work, arrays works in python. I have been through the videos and multiple study materials shared during the session to understand its working. 

## How were these changes tested?
     I have tried to run all the commands and scripts in Visual Studio

## A reference to a related issue in your repository (if applicable)

## Checklist
- [ Y] I can confirm that my changes are working as intended

Commit:
Updated Assignment2 as a part of Python Session

anjali-deshpande-hub reviewed (changes requested):
Question 1: Replace the hardcoded absolute paths to relative paths so that a Learning support can run the code successfully as well. For example: change `"/Users/bmanna/desktop/diswrk1/python/05_src/data/assignment_2_data/inflammation_01.csv"` to `"../../05_src/data/assignment_2_data/inflammation_01.csv"`

Although you have printed the contents of the file correctly, the requirement is to use readlines/readline method to read the csv file into a variable and then use the variable in a for loop to print each row.  Please make the appropriate changes.

Question 3:   Perhaps you want to rename the variable data_min to data_mean since its the mean value.

I noticed that you sometimes leave space between the function name and '('  

`print (recount)`
`print( filepath,"has error in patient data ?", file_has_error)`

Sometimes, there is no space before and after = in some statements: `file_has_error=detect_problems( (filepath))`

The indent is double tab where it can be single. Eg: for loop for testing.

It's best to follow a consistent style throughout the code. PEP 8 (https://peps.python.org/pep-0008/) is a great reference for this.

Commit:
Updated Assignment2 as a part of Python Session

Bishudsi commented:
Hi @anjali-deshpande-hub ,
Thank you for your reviewing and the feed back comments. I really helpful. I have tried to incorporate the feedback comments and I have pushed the updates again into git. Request you to please review the changes again and let me know if the updates look fine or for any other observations.
1. corrected the Relative path 
2. corrected the spacing while using functions or around "=".

anjali-deshpande-hub reviewed (approved):
Good! 
Just a quick heads-up - there’s still some double tab spacing in the for loop code in Question 3. No biggie though. I’ll go ahead and mark the PR as approved. 

The small details like consistent indentation, spacing, formatting, and other code style habits might feel a bit tricky or easy to overlook at first. But it will eventually become automatic for you as you keep coding more.

Bishudsi commented:
Thank you @anjali-deshpande-hub , for the review comments and feedback. it's really helpful I will surely incorporate the feedback and suggestions and get into a habit of using it consistently.
